package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@microsoft/vscode-serial-monitor-api",
+  "version": "0.1.0",
```

Comment:
I'm wondering if we should start the `Version` enum at `v1` instead. Once we're confident that the pipelines and API work the way we want them to, we can bump the `package.json` version number to `1.0.0`.

Reply:
Discussed offline. We'll keep `v0` and then change to `v1` once we have it stable.